src/v/rpc/rpc_server.cc

```diff
@@ -104,7 +104,13 @@ rpc_server::send_reply(ss::lw_shared_ptr<server_context_impl> ctx, netbuf buf) {
 
 ss::future<> rpc_server::send_reply_skip_payload(
   ss::lw_shared_ptr<server_context_impl> ctx, netbuf buf) {
-    co_await ctx->conn->input().skip(ctx->get_header().payload_size);
+    try {
+        co_await ctx->conn->input().skip(ctx->get_header().payload_size);
```

Comment:
nit: could you add a comment that we might expect an exception here if the connection is closed? Otherwise it's not immediately obvious why this is necessary

Reply:
Good catch, will add

Reply:
i think it might make sense to handle this back in `rpc_server::dispatch_method_once` ?